README.md

```diff
@@ -0,0 +1,52 @@
+Yocto Layer for Crosswalk
+============================
+
+This layer adds the packages necessary for adding support for [Crosswalk Project](https://crosswalk-project.org/) to your Yocto distribution.
+
+## Supported Yocto Configurations
+* [Ostro™ OS XT](https://github.com/ostroproject/ostro-os-xt)
+
+Usages
+=====
+### Ostro™ OS XT
+#### Build Ostro™ OS XT image
+Please refer to [Building Ostro™ OS XT](https://github.com/ostroproject/ostro-os-xt#building-ostro-tm-os-xt) to setup and build the base Ostro™ OS XT image, assuming the `ostro-os-xt` is checked out at `/path/to/ostro-os-xt` folder.
+
+#### Build Ostro™ OS XT image with Crosswalk layer
+Checkout the `meta-crosswalk` to local folder, assuming `/path/to/meta-crosswalk`.
+
+Add the `meta-crosswalk` layer to `/path/to/ostro-os-xt/build/conf/bblayers.conf`
+```bitbake
+OSTRO_LAYERS += "/path/to/meta-crosswalk"
+```
+
+In `/path/to/ostro-os-xt/build/conf/local.conf`:
+
+Add Crosswalk security flags configuration include file by adding
+```bitbake
+require /path/to/meta-crosswalk/conf/distro/include/ostro-xt-security-flags.inc
+```
+
+Add Crosswalk dependent recipes as supported recipes by adding
+```bitbake
+SUPPORTED_RECIPES_append = "/path/to/meta-crosswalk/conf/distro/include/ostro-xt-supported-recipes.txt"
+```
+
+Install Crosswalk into Ostro™ OS XT image by adding
+```bitbake
+OSTRO_IMAGE_EXTRA_INSTALL_append = "crosswalk"
```

Comment:
This is the variable used in Ostro OS. For Ostro OS XT, I think `OSTRO_XT_IMAGE_EXTRA_INSTALL` is the recommended one. Also, there should be a whitespace before `crosswalk`.